pipes/WDL/workflows/tasks/ncbi.wdl

```diff
@@ -87,12 +96,13 @@ task prepare_genbank {
   Array[File]+ assemblies_fasta
   Array[File]+ annotations_tbl
   File         authors_sbt
-  File?        coverage_table # summary.assembly.txt (from Snakemake)
-  File?        genbankSourceTable
-  File?        biosampleMap
-  String?      sequencingTech
-  String?      comment
-  String       out_prefix = "ncbi_package"
+  File         biosampleMap
+  File         genbankSourceTable
+  File?        coverage_table # summary.assembly.txt (from Snakemake) -- change this to accept a list of mapped bam files and we can create this table ourselves
+  String       sequencingTech
+  String       comment # TO DO: make this optional
```

Comment:
What is blocking this from being optional, or having a default placeholder value?

Reply:
I had a lot of difficulty with the WDL syntax of specifying an optional parameter with a double-quoted string as a value (just because it passes womtool validate doesn't mean dxWDL and Cromwell agree about whether they like the syntax)... gave up and just made it a mandatory field for now since it's almost always specified anyway in a submission. Double quoting is important because people are putting sentences with spaces and punctuation here.

Reply:
Could make it an optionl text file